src/pages/legacy/home/index.tsx

```diff
@@ -71,7 +80,9 @@ function Home() {
                       </div>
                     </div>
                   </td>
-                  <td className="text-right">${(pool.reserveX * prices[pool.tokenX.symbol] + pool.reserveY * prices[pool.tokenY.symbol]).toLocaleString(undefined)}</td>
+                  <td className="text-right">
+                  ${(pool.reserveX * prices[pool.tokenX.symbol] + pool.reserveY * prices[pool.tokenY.symbol]).toLocaleString(undefined)}
```

Comment:
put this in a variable above the return?

Reply:
Its a bit more complex, the state context isn't strictly typed and we have to build out error boundaries.  the app wont build due to page type-errors, but it also crashes outside those views due to lack of error mgmt.   matt and I are going to be working thru some refactors to reflect the base-app, this pr will change a lot over the next day.

Reply:
previously, react would just steamroll thru those crashes but we need stability